zdm/scripts/run_mcmc.slurm

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+#SBATCH --job-name=CRAFT
+#SBATCH --output=../mcmc/Hoffmann2023_CRAFT_no_191228.out
+#SBATCH --ntasks=20
+#SBATCH --time=24:00:00
+#SBATCH --export=NONE
+# SBATCH --mem=50GB
+#SBATCH --mem-per-cpu=3GB
+
+# source /fred/oz002/jhoffmann/FRB_library/setup.sh
+# source /fred/oz002/jhoffmann/FRB_library/loadzdm.sh
+
+source $ZDM/.zdm_env/bin/activate
+
+cd $ZDM/zdm
+
+# outfile="Hoffmann2023_CRAFT_no_191228"
+outfile="DSA"
+walkers=40
+steps=1000
+
+surveys="DSA3.ecsv" #"parkes_mb_class_I_and_II.ecsv" # DSA3.ecsv FAST.ecsv CRAFT_class_I_and_II.ecsv private_CRAFT_ICS_892.ecsv private_CRAFT_ICS_1300.ecsv private_CRAFT_ICS_1632.ecsv 
```

Comment:
Just to be sure: this current implementation makes it look like *only* DSA is being modelled by default. Can you make sure the version being committed has "everything" as the default behaviour, and other, commented alternatives have notes like "used for HOFFMAN2023" or similar? Or if there are old junk options here you'd never revisit, just delete them?